app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateSimpleProductReassignSource.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateSimpleProductReassignSource">
+        <annotations>
+            <stories value="Add Simple Product"/>
+            <title value="Simple Product created and source can be reassigned"/>
+            <description value="You should be able to create a New Product via the Admin, then reassign previously assigned source."/>
+            <group value="msi"/>
+            <group value="multi_mode"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <amOnPage url="{{AdminProductIndexPage.url}}" stepKey="navigateToProductGridPage"/>
+            <waitForPageLoad time="30" stepKey="waitForProductGridPageLoad"/>
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logoutOfAdmin"/>
+        </after>
+
+        <click selector="{{AdminProductGridActionSection.addProductToggle}}" stepKey="clickAddProductDropDown"/>
+        <click selector="{{AdminProductGridActionSection.addSimpleProduct}}" stepKey="clickAddSimpleProduct"/>
+
+        <actionGroup ref="AssignSourceToProductActionGroup" stepKey="assignSource">
+            <argument name="sourceCode" value="default"/>
```

Comment:
Instead of hardcoding I suggest to use a [data entity for default source](https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/InventoryAdminUi/Test/Mftf/Data/MsiSourceData.xml#L11-L24).
Please use this notation `value="_defaultSource.source_code"` if argument type is not specified in ActionGroup or use this notation `value="{{_defaultSource.source_code}}"` if argument type specified.

Also apply same for line 36 and 40